tests/unit/test_main.py

```diff
@@ -32,10 +34,39 @@ def test_load_config(tmpdir, suffix, config_file, loaded_config):
     conf_file = tmpdir.mkdir('config').join(filename)
     conf_file.write(config_file)
     config = main._load_config(root=conf_file.dirpath())
-
     assert loaded_config == config
 
 
+def test_load_config_deep_merges(tmpdir, config_file, loaded_config):
+    """Additively merge user file to main config."""
+    config_dir = tmpdir.mkdir('mergeconfig')
+    main_conf_file = config_dir.join('gordon-janitor.toml')
+    main_conf_file.write(config_file)
+
+    user_conf_file = config_dir.join('gordon-janitor-user.toml')
+    user_conf_file.write('[core.logging]\nlevel = "error"\n')
+    config = main._load_config(root=config_dir.strpath)
```

Comment:
Might want to rename this 'actual_config', and put a blank line above it and remove the blank at 45.